src/main/java/se/uu/elephas/services/UserServiceImpl.java

```diff
@@ -37,12 +39,25 @@ public Object create(User user, MessageDigest md) {
         String token = bytes.toString();
         user.setToken(token);
 
-        return (userRepository.save(user));
 
+        try {
```

Comment:
i think it's better to catch this exception inside the controller and then to return the proper message inside the httpresponse